.github/workflows/pr-title.yml

```diff
@@ -0,0 +1,68 @@
+name: Check Conventional Commits format
+
+on:
+  pull_request_target:
```

Comment:
`pull_request_target` will use the workflow config from `main` instead of the PR's, so PRs by external contributors do not need pre-approval to run this job (in contrast with `pull_request`).